src/main/play-doc/operation/ClusterTroubleshooting.md

```diff
@@ -0,0 +1,54 @@
+# Cluster Troubleshooting
+
+Cluster partition is a failure that may occur when running a cluster based application. To recover from this situation ConductR utilizes Akka SBR (Split Brain Recovery). Akka SBR is a part of [Akka Typesafe Reactive Platform](http://www.typesafe.com/products/typesafe-reactive-platform) version `15v01p05`.
+
+The Akka SBR strategy adopted by ConductR is `keep-majority`.
+
+The `keep-majority` strategy ensures when cluster split occurs:
+* The partition which has the majority of the nodes are kept running
+* Other partition(s) which has less than majority of the nodes will have their ConductR process terminated.
+
+## Cluster Split Recovery
+
+Upon cluster split, Akka SBR will terminate the ConductR processes within the partition that does not have majority number of nodes.
+
+**Recovery Steps**
+* Obtain the list of the nodes where ConductR process have been terminated.
+* Restart the ConductR process on one of the nodes in the list.
+```bash
+sudo service conductr restart
+```
+* Upon restart, the ConductR processes in these nodes will attempt to reconnect to last known members of the cluster based on the information stored in the `seed-nodes` file. This will effectively allow the restarted node to rejoin the cluster.
+* If the service has been restarted successfully continue with the next node and repeat that until all services has been restarted.
+
+**Notes on `seed-nodes` file**
+* The address of last known members in the cluster are kept in what we call seed nodes file.
+* The file is located at `/usr/share/conductr/conf/seed-nodes`.
+* By default ConductR keeps the latest 3 reachable members in the cluster in the seed nodes file.
+
+## Whole Cluster Termination Recovery
+
+Should the cluster split occurs in such way that each partition does not have majority of the node, all ConductR process within the cluster will be terminated.
```

Comment:
Maybe link to ClusterSetupConsiderations i.e. Deploying a cluster across at least 3 network partitions improves the chances of maintaining a majority and preventing cluster shutdown. See ClusterSetupConsiderations.md for more information.

also "split occurs" -> "split  occur"